src/widgets/component-new-edit/components/variables/collected-variables.jsx

```diff
@@ -99,6 +99,8 @@ class CollectedVariables extends Component {
       componentName,
       responseFormatType,
     } = this.props;
+    const hiddenCollected = responseFormatType !== "TABLE";
```

Comment:
You should use the const we already defined in the application

Reply:
Changed.

Reply:
Done!